src/components/UserStats/Base.vue

```diff
@@ -63,7 +63,7 @@ export default {
   methods: {
     ...mapActions(['lazyGetNFTPurchaseInfo', 'lazyGetNFTOwners']),
     lazyGetAllCollectedPurchaseInfo() {
-      const classIdSet = new Set(this.collectedItems.map(n => n.classId));
+      const classIdSet = new Set(this.collectedClassIds.map(id => id));
```

Comment:
Seems no need mapping 
```suggestion
      const classIdSet = new Set(this.collectedClassIds);
```